test/srt/openai/test_serving_completions.py

```diff
@@ -13,12 +13,7 @@
     CompletionStreamResponse,
     ErrorResponse,
 )
-from sglang.srt.entrypoints.openai.serving_completions import CompletionHandler
-from sglang.srt.entrypoints.openai.utils import (
-    build_base_sampling_params,
-    create_error_response,
-    create_streaming_error_response,
-)
+from sglang.srt.entrypoints.openai.serving_completions import OpenAIServingCompletion
 from sglang.srt.managers.io_struct import GenerateReqInput
 from sglang.srt.managers.tokenizer_manager import TokenizerManager
```

Comment:
Let's remove some test classes that will likely to be refactored in next week's refactor:

`TestEchoHandling`: `_get_echo_text()`'s existance in the next refactor is unclear.
`TestPromptHandling`: many of its tests are kinda overlapped with `TestCompletionRequestConversion`. We can keep one for now. 
`TestAsyncMethods`
`TestEdgeMethods`

I think the overall test structure is a bit complex here. For instance, `TestResponseBuilding`, `TestCompatibilityWithAdapter`, `TestPromptHandling`, and etc they all call _convert_to_internal_request, so it makes more sense to put them into a single class.

We can refine this later. For now I feel it would be better to remove the above test classes.